bin/fin

```diff
@@ -104,9 +104,11 @@ DOCKSAL_ENVIRONMENT="${DOCKSAL_ENVIRONMENT:-local}"
 # Network settings
 ## Docker for Mac/Win
 DOCKSAL_NATIVE_IP="192.168.64.100"
+DOCKSAL_NATIVE_HOST_IP="192.168.64.1"
```

Comment:
What is the difference between DOCKSAL_DEFAULT_HOST_IP and DOCKSAL_NATIVE_HOST_IP?

Reply:
There is no difference at the moment. There may not be going forward, in which case, we will blend the two. I just don't want to do this just yet.